geonode/tasks/tasks.py

```diff
@@ -204,3 +203,9 @@ def set_permissions(self, permissions_names, resources_names,
                 delete_flag,
                 verbose=True
             )
+
+
+settings.CELERY_BEAT_SCHEDULE["delete-incomplete-session-resources"] = {
```

Comment:
It does not make much sense to put this setting here.

Either:
1. We consider the `geonode.upload` module as a plugin, in that case the task should be defined on the `geonode.upload` module itself
2. We consider `geonode.upload` module as part of the cose, in that case let's harmonize with the others and place this into `geonode.settings`

Reply:
I myself think, as suggested before, all the schedules should be at one place. But our `settings.py` is already over 2100 line. My humble suggestion is, bring even the other two settings in the standard place, `tasks`.

- First and foremost that is the expected place for this.
- We may not be able to adjust all future schedules in settings.py without more mess.
- Thirdly, even in `settings.py` they will not be at one place, if we see the way harvester is brought in. It has its own conditional place.
- Another option is not use `CELERY_BEAT_SCHEDULE` at all in the settings, rather use Celery's `app.config.beat_schedule`, so that they are not scattered. And we don't need to override settings at all. At this point, these are only a couple and we can do it easily.

Reply:
Nope, this will create confusion to people already using `settings` and also they would need to update their instances.

I prefer moving just this one to `geonode.upload.tasks`.

Reply:
Let's go ahead for this one then.